back/src/mailer/templates/renderers.ts

```diff
@@ -27,12 +33,15 @@ export function mustacheRenderer(filename: string) {
   return render;
 }
 
-type MailRendererInput<V> = {
-  to: Recipient[];
+export type MailRendererInput<V> = {
   cc?: Recipient[];
   attachment?: Attachment;
   variables?: V;
-};
+} & (
```

Comment:
J'ai un peu galéré pour le typing ici, puisque `to` OU `messageVersions` doit être obligatoire (mais pas les deux en même temps)

Reply:
Clever